backend/core/src/secondary-adapters/postgres/channel/channel-repository-adapter.ts

```diff
@@ -11,6 +11,107 @@ export class ChannelRepositoryAdapter implements ChannelRepository {
     this.prismaClient = prismaClient;
   }
 
+  async addSubscription(userId: string, channelId: string): Promise<CreateSubscriptionResponseDto | null> {
+    await this.prismaClient.channel.update({
+      where: {
+        id: channelId,
+      },
+      data: {
+        subscriptions: {
+          createMany: {
```

Comment:
why createMany? there's only one new sub

Reply:
done, change to create